### ISSUE TITLE
PoseTrack18 results?

### ISSUE THREAD

**Issue by ybai62868** [mks0601/TF-SimpleHumanPose#1]
Hi, you write very good code and recently I am working on PoseTrack (including 3 tasks) too.
I use the msra's pytorch code (simple baseline) finetune the PoseTrack 2018 dataset  and all parameters I use according to the paper. 
Using gt_box on PoseTrack2018 dataset for Human Pose Estimation, I get the results which is worse than you reported:
my eval results:

Loading data
('# gt frames  :', 3902)
('# pred frames:', 3902)
Evaluation of per-frame multi-person pose estimation
('saving results to', './out/total_AP_metrics.json')
Average Precision (AP) metric:
& Head & Shou & Elb  & Wri  & Hip  & Knee & Ankl & Total\\
& 49.4 & 89.2 & 83.9 & 76.1 & 81.0 & 80.3 & 74.6 & 74.6 \\

Can you teach me how to finetune the model and I want to get the same results as you!
Thank you!
BTW, do you plan to implement the tracking code in this paper(simple baseline for human pose estimation and tracking)? I am doing this section ...

**Comment by mks0601** [comment]
Sounds good!

**Comment by lxtGH** [comment]
@mks0601 Hi!!  pose track dataset have 15 joints(2 joints are not annotated). How did you handle this?

**Comment by mks0601** [comment]
Just use pre-trained model on COCO, and set loss zero for not annotated l_ear and r_ear.

**Comment by Vhunon** [comment]
Where did you set the loss to zero in the code?

**Comment by mks0601** [comment]
I can't remember clearly, but 
Please check L86 of `main/gen_batch.py`.
joints[:,2] of those joints would be zero.

**Comment by Vhunon** [comment]
I couldn't figure out how to set these 2 keypoints to 0 loss.
But I think its this [line](https://github.com/mks0601/TF-SimpleHumanPose/blob/099b9dbfde38aded0f11728de4719e0cd75d649f/main/model.py#L76)? 
Also here:
loss = tf.reduce_mean(tf.square(heatmap_outs - gt_heatmap) * valid_mask)

**Comment by mks0601** [comment]
I think so. The valid mask is generated from joints[:,2]. The mask values would be zero for not annotated joints.
